Index: task/module_duiwu/module_duiwu.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from ocr.main import ocr_txt_click, ocr_txt\r\nfrom task.module_duiwu.address_area import list_name_area, list_click_area\r\nfrom tools.reg_list_name import reg_list_name\r\n\r\n\r\ndef module_ocr_duiwu_name(device, path, is_main=False):\r\n    area = list_name_area,\r\n    if is_main:\r\n        area = list_click_area\r\n    device.screenshot().crop(area).save(path)\r\n    result = ocr_txt(path)\r\n    return reg_list_name(result)\r\n\r\n\r\ndef module_click_duiwu(device, path, auto_txt, is_main=False):\r\n    area = list_name_area\r\n    if is_main:\r\n        area = list_click_area\r\n    device.screenshot().save(path)\r\n    return ocr_txt_click(path, auto_txt, '', area, True)\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/task/module_duiwu/module_duiwu.py b/task/module_duiwu/module_duiwu.py
--- a/task/module_duiwu/module_duiwu.py	(revision bb204bade62c1eccfd1bece1324e977468c7cd88)
+++ b/task/module_duiwu/module_duiwu.py	(date 1692017911108)
@@ -18,3 +18,5 @@
         area = list_click_area
     device.screenshot().save(path)
     return ocr_txt_click(path, auto_txt, '', area, True)
+
+
