Index: dispatcher/main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from apscheduler.schedulers.blocking import BlockingScheduler\r\n\r\n\r\ndef my_task():\r\n    print(\"Hello, world!\")\r\n    scheduler.add_job(my_task2, 'interval', seconds=2)\r\ndef my_task2():\r\n    print(\"222333\")\r\n\r\n\r\nscheduler = BlockingScheduler()\r\nif __name__ == '__main__':\r\n    scheduler.add_job(my_task, 'interval', seconds=2)  # 每隔 5 秒执行一次任务\r\n    scheduler.start()\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/dispatcher/main.py b/dispatcher/main.py
--- a/dispatcher/main.py	(revision 4d844cbdcca482fb11e35624ba9b3322282eb58f)
+++ b/dispatcher/main.py	(date 1692007765591)
@@ -4,6 +4,8 @@
 def my_task():
     print("Hello, world!")
     scheduler.add_job(my_task2, 'interval', seconds=2)
+
+
 def my_task2():
     print("222333")
 
@@ -12,3 +14,4 @@
 if __name__ == '__main__':
     scheduler.add_job(my_task, 'interval', seconds=2)  # 每隔 5 秒执行一次任务
     scheduler.start()
+    scheduler.remove_job('my_task')
